index-es.html

```diff
@@ -71,9 +71,32 @@
       sample_subm_lab: "Laboratorio de sometimiento",
       sample_authors: "Autores",
 
+      // walkthrough tour
```

Comment:
@horaciobam - can you please have a look at this HTML file?

Reply:
Can we say it is the lab from which the sample is from? I think we don't have a proper way to translate "Submitting lab", because "de sometimiento" implies that people submit their samples there, and not the other way around.